test/specs/TC1_Login_valid_user.js

```diff
@@ -0,0 +1,21 @@
+
+import loginPage from '../pageobjects/login.page.js'
+import inventoryPage from '../pageobjects/inventory.page.js'
+import { config } from '../../config.js';
+
+const { username, password } = config.valid_user;
+
+describe('TC1 - Login valid user test', () => {
```

Comment:
@karenn32 Not bad idea for the POM model, same used on the Rentzila. But, it must be a pit modified. I'll call you about that.